routes/sites.js

```diff
@@ -85,7 +89,7 @@ router.get('/', async function(req, res, next) {
 
 
       siteNames= siteNames.concat(isomerRepos)
```

Comment:
put the space between the variable and the assignment operator please `siteNames=`

Reply:
resolve in #44